apps/omg/lib/omg/typed_data_hash/config.ex

```diff
@@ -22,23 +22,31 @@ defmodule OMG.TypedDataHash.Config do
   alias OMG.TypedDataHash.Tools
 
   # Needed for test only to have value of address when `:contract_address` is not set
-  @fallback_ari_network_address "0x44de0ec539b8c4a4b530c78620fe8320167f2f74"
+  @fallback_contract_addr <<1::size(20)-unit(8)>>
```

Comment:
could you help me understand why this is needed? I'm sensing there might be a better way of doing this...

Reply:
yeah, I scratched my head a bit here too, finding a better way. So normally we want to use the address from the config. But if it's not there (e.g. in a unit test, but not in an integration test) we want just anything.

I tried to put the fallback addr into `config/test.exs` but something wasn't clicking.

I can revisit this now, having all the other stuff more or less behind.